bundle/src/files.rs

```diff
@@ -126,10 +126,15 @@ impl FileSet {
 
             // Save file under junit/0, junit/1, etc.
             // This is to avoid having to deal with potential file name collisions.
+            let path_formatted = match original_path_abs.ends_with(".xml") {
+                // we currently support junit and internal binary files
+                true => format!("junit/{}", file_counter.count_file()),
+                false => format!("bin/{}", file_counter.count_file()),
```

Comment:
I was of the impression we would have to do a lot more than this PR does to support a new file format, i.e. handle it in `extract_failed_tests`, etc.

Reply:
There's a change in the mono repo. But it wasn't that much to get started.